scripts/bb-test-zfstests.sh

```diff
@@ -52,16 +52,9 @@ $ZFSTESTS $TEST_ZFSTESTS_OPTIONS \
     -d $TEST_ZFSTESTS_DIR \
     -s $TEST_ZFSTESTS_DISKSIZE \
     -r $TEST_ZFSTESTS_RUNFILE &
-CHILD=$!
-
-sleep 1
-TEST_LOG=$(ls -t /var/tmp/test_results/*/log | head -1)
-rm -f log
-ln -s $TEST_LOG log
-
-wait $CHILD
```

Comment:
How does this work?  Buildbot still expects to find the log file relative to its current working directory which will be some thing like `/var/lib/buildbot/slaves/zfs/<slave-name>/build/tests/zfstests`.  But the `current` symlink should be created in `/var/tmp/test_results/current`.  We still need to keep that `ln` but link it to the known log path.

```
ln -s /var/tmp/test_results/current/log log
```